pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -51,6 +51,28 @@ static __always_inline void update_conn_stats(conn_tuple_t *t, size_t sent_bytes
         return;
     }
 
+    protocol_t local_protocol = PROTOCOL_UNCLASSIFIED;
```

Comment:
This code doesn't seem to be touching any connection stats, does it need to be here? It would also add an entry into the `conn_stats` map when it may not be needed. Can you just put this in a helper and call that from the `tcp_recvmsg` probes?

Reply:
The `get_conn_stats` call above will add an entry if it does not already exist in `conn_stats`

Reply:
Also, if you call `handle_message` from the `tcp_recvmsg` probe, you'll duplicate what's happening in the probe for `tcp_cleanup_rbuf`. Why didn't you add the call there?

Reply:
The idea is to reduce the chances for races between the socket filter and kprobes

Is there a way to keep tcp_recvmsg  without duplicating the data?
And furthermore, why does it duplicate the data if all values are zero (or default) and should not affect any current field in the conn_stat message?

Reply:
will do that change !